components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -247,35 +246,53 @@ public DataSection phenotypeBody(Patient patient) throws Exception
         if (sortedFeatures.size() == 0) {
             // offset is included to account for the presence of both "positive" and "negative" in "present"
             int offset = bothTypes ? 1 : 0;
-            Integer emptyX = 0;
             for (int i = 0; i < present.size() - offset; i++) {
-                DataCell cell = new DataCell("", emptyX, 0);
+                DataCell cell = new DataCell("", i, 0);
                 section.addCell(cell);
-                emptyX++;
             }
         }
         // section.finalizeToMatrix();
         return section;
     }
 
+    public void variantsSetup(Set<String> enabledFields) throws Exception
+    {
+        String sectionName = "variants";
+        String[] fieldIds =
+        { "variants", "variants_protein", "variants_transcript", "variants_dbsnp", "variants_zygosity",
+            "variants_effect",
+            "variants_interpretation", "variants_inheritance", "variants_evidence", "variants_segregation",
+        "variants_sanger" };
+        // FIXME These will not work properly in different configurations
+        String[][] headerIds =
+        {
+            { "cdna" },
+            { "protein", "cdna" },
+            { "transcript", "protein", "cdna" },
+            { "dbsnp", "transcript", "protein", "cdna" },
+            { "zygosity", "dbsnp", "transcript", "protein", "cdna" },
+            { "effect", "zygosity", "dbsnp", "transcript", "protein", "cdna" },
+            { "interpretation", "effect", "zygosity", "dbsnp", "transcript", "protein", "cdna" },
+            { "inheritance", "interpretation", "effect", "zygosity", "dbsnp", "transcript", "protein", "cdna" },
+            { "evidence", "inheritance", "interpretation", "effect", "zygosity", "dbsnp", "transcript", "protein",
+            "cdna" },
+            { "segregation", "evidence", "inheritance", "interpretation", "effect", "zygosity", "dbsnp", "transcript",
+                "protein", "cdna" },
+                { "sanger", "segregation", "evidence", "inheritance", "interpretation", "effect", "zygosity", "dbsnp",
+                    "transcript", "protein", "cdna" } };
+        Set<String> present = addHeaders(fieldIds, headerIds, enabledFields);
+        this.enabledHeaderIdsBySection.put(sectionName, present);
+    }
+
     public void genesSetup(Set<String> enabledFields) throws Exception
     {
         String sectionName = "genes";
-        String[] fieldIds = { "genes", "genes_comments", "rejectedGenes", "rejectedGenes_comments" };
+        String[] fieldIds = { "genes", "genes_status", "genes_strategy", "genes_comments" };
```

Comment:
Same here, the format of the excel export is changed, which is different from making it translatable. If Some users tell us their scripts broke because column values change it would be harder to track the commit that did it by looking at commit names